crates/bevy_reflect/src/list.rs

```diff
@@ -95,7 +95,7 @@ pub trait List: PartialReflect {
     fn iter(&self) -> ListIter;
 
     /// Drain the elements of this list to get a vector of owned values.
```

Comment:
Nit: We should probably clarify for those unfamiliar with "drain" that this will leave the list empty.

And perhaps it would be good to also mention that the elements will maintain their order.

Reply:
Done.